src/snowflake/connector/file_transfer_agent.py

```diff
@@ -348,14 +357,15 @@ def execute(self):
     def upload(self, large_file_metas: List['SnowflakeFileMeta'], small_file_metas: List[SnowflakeFileMeta]):
         storage_client = SnowflakeFileTransferAgent.get_storage_client(
             self._stage_location_type)
-        client = storage_client.create_client(
-            self._stage_info,
-            use_accelerate_endpoint=self._use_accelerate_endpoint
-        )
+        client_meta = SFResourceMeta(storage_client, self._stage_info, self._use_accelerate_endpoint)
+        # client = storage_client.create_client(
+        #    self._stage_info,
+        #    use_accelerate_endpoint=self._use_accelerate_endpoint
+        # )
```

Comment:
Now everything else will still use `client` except this parallel uploading (this uses `cln_meta`). Either change all of them, or non of them, the file meta class is already bulky

Reply:
Also please don't leave commented out code there